cms/tests/frontend/integration/helpers/cms.js

```diff
@@ -32,27 +32,31 @@ module.exports = function (casperjs, settings) {
         /**
          * removes a page by name, or the first encountered one
          *
-         * @public
          * @param {Object} opts
          * @param {String} [opts.title] Name of the page to delete
          */
         removePage: function (opts) {
             var that = this;
             return function () {
                 return this.thenOpen(globals.adminPagesUrl)
-                    .waitUntilVisible('.cms-pagetree-jstree [href*="delete"]')
+                    .waitUntilVisible('.js-cms-pagetree-options')
                     .then(that.expandPageTree())
                     .then(function () {
                         var pageId;
                         if (opts && opts.title) {
                             pageId = that.getPageId(opts.title);
                         }
 
-                        if (pageId) {
-                            this.click('.cms-pagetree-jstree [href*="delete"][href*="' + pageId + '"]');
-                        } else {
-                            this.click('.cms-pagetree-jstree [href*="delete"]'); // first one
-                        }
+                        var data = pageId ? '[data-id="' + pageId + '"]' : '';
+                        var href = pageId ? '[href*="' + pageId + '"]' : '';
```

Comment:
skip) i plan to integrate eslint in the near future and then will readjust the rules.